src/woo_publications/api/openapi.yaml

```diff
@@ -987,7 +999,6 @@ paths:
           type: array
           items:
             type: string
-            format: uuid
```

Comment:
Because we no longer use __uuid in for the field_name the format doesn't get set right. But it is described in the help_text that it should be a UUID.